website/blog/2022/04/17/api-observability.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "API Observability with Apache APISIX Plugins"
+keywords:
+- Apache APISIX
+- API Gateway
+- Observability
+authors:
+  - name: "Boburmirzo"
+    title: "Author"
+    url: "https://github.com/Boburmirzo"
+    image_url: "https://avatars.githubusercontent.com/u/14247607?v=4"
+description: In this blog post, we can leverage the power of some Apache APISIX Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
+---
+
+> In this blog post, we can leverage the power of some Apache APISIX Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact APISIX's users.
+
+<!--truncate-->
+
+![Cover image for API Observability with Apache APISIX Plugins](https://static.apiseven.com/202108/1650506677636-f4d1ffa0-d848-4264-b497-de1061da0faf.png)
+
+In this blog post, we can leverage the power of some Apache APISIX Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
```

Comment:
```suggestion
> In this blog post, we can leverage the power of some Apache APISIX Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact APISIX's users.

<!--truncate-->

![Cover image for API Observability with Apache APISIX Plugins](https://static.apiseven.com/202108/1650506677636-f4d1ffa0-d848-4264-b497-de1061da0faf.png)

```

Reply:
cause this part are same,  I think maybe we could only just remain the above part
<img width="808" alt="image" src="https://user-images.githubusercontent.com/39793568/165036527-855a0333-e874-42e6-8cd7-cabfc5e20e07.png">

Reply:
Ok, I noticed this part at the time, but I was not sure whether to delete it.